City-of-Pori/pori-kada
# PORI-316 video for attraction card

To test:

1. ./build.sh update
2. drush updb -y && drush fra -y && drush cc all
3. Go to any attraction card on local.visitpori.fi and edit it.
4. Make sure you can see a Video tab. There you should have a field where you can paste youtube or vimeo video url. Do so.
5. Save the page and make sure that the video is displayed under the main text content.
